etc/ProdOfflineConfiguration.py

```diff
@@ -113,23 +113,22 @@
 hiTestppScenario = "ppEra_Run3"
 
 # Defaults for processing version
-defaultProcVersionRAW = 1
+defaultProcVersionRAW = 2
 alcarawProcVersion = {
-    'default': "1"
+    'default': "2"
 }
 
 defaultProcVersionReco = {
-    'default': "1"
+    'default': "2"
 }
 
 expressProcVersion = {
-    'default': "1"
+    'default': "2"
 }
 
 # Defaults for GlobalTag
-expressGlobalTag = "126X_dataRun3_Express_v3"
-promptrecoGlobalTag = "126X_dataRun3_Prompt_v3"
-alcap0GlobalTag = "126X_dataRun3_Prompt_v3"
```

Comment:
@germanfgv is this dropped on purpose?

Reply:
Yes, it was on porpuse. `alcap0GlobalTag` is not being used anymore (has not been used for years, really).

Reply:
ok, thanks

Reply:
yes indeed you are right @germanfgv , by searching in github this only appears in the Run1 configuration 😄 sorry for the noise!